internal/mfaportal/resources/frontend/src/api/index.ts

```diff
@@ -13,3 +13,8 @@ export function logout(): Promise<boolean> {
     return (res.status === 204)
   });
 }
+export function login(): Promise<boolean> {
+  return client.post("/api/login").then((res) => {
```

Comment:
The `/api/login` route should not exist it breaks the security properties of wag, plus it doesnt work with the currently designed authenticators package.